gulpfile.js

```diff
@@ -863,6 +895,8 @@ exports.skillmap = skillmap;
 exports.authcode = authcode;
 exports.multiplayer = multiplayer;
 exports.kiosk = kiosk;
+exports.teacherTool = teacherTool;
+exports.tt = teacherTool;
```

Comment:
Is this safe to do? Just wanted to double check since it looks like all other exports are set to something different.

Reply:
I believe so. It just means both "gulp teacherTool" and "gulp tt" will run the same command. (At least, as far as I'm aware.)